packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +21,144 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class allows several methods to lazily provide children, which is more
+/// efficient when there are too many or infinite children, by bringing in
+/// only the children that are visible in viewport, and remove them when they
+/// are no longer visible.
+///
+/// See also:
+///  * [ListWheelChildListDelegate], a delegate that supplies children using an
+///    explicit list.
+///  * [ListWheelChildLoopingListDelegate], a delegate that supplies infinite
+///    children by looping an explicit list.
+///  * [ListWheelChildBuilderDelegate], a delegate that supplies children using
+///    a builder callback.
+abstract class ListWheelChildDelegate {
+  /// Return the child at the given index. If the child at the given
+  /// index does not exist, return null.
+  Widget build(BuildContext context, int index);
+
+  /// Returns an estimate of the number of children this delegate will build.
+  int get estimatedChildCount;
+
+  /// Returns the true index for a child built at a given index. Defaults to
+  /// the given index, however if the delegate is [ListWheelChildLoopingListDelegate],
+  /// this value is the index of the true element that the delegate is looping to.
+  int trueIndexOf(int index) => index;
+
+  /// Called to check whether this and the old delegate are actually 'different',
+  /// so that the caller can decide to rebuild or not.
+  bool shouldRebuild(covariant ListWheelChildDelegate oldDelegate);
+}
+
+/// Class that supplies children using an explicit list. Children are provided
+/// in the range [0, list.length - 1], outside of that range a null child will
+/// be returned.
+class ListWheelChildListDelegate extends ListWheelChildDelegate {
+  /// Construct the delegate from a concrete list of children.
+  ListWheelChildListDelegate({@required this.children}) : assert(children != null);
+
+  /// The list containing all children that can be supply.
+  final List<Widget> children;
+
+  @override
+  int get estimatedChildCount => children.length;
+
+  @override
+  Widget build(BuildContext context, int index) {
+    if (index < 0 || index >= children.length)
+      return null;
+    return children[index];
+  }
+
+  @override
+  bool shouldRebuild(covariant ListWheelChildListDelegate oldDelegate) {
+    return children != oldDelegate.children;
+  }
+}
+
+/// A delegate that supplies an infinite amount of children from an explicit
```

Comment:
Be sure to explain in this first line of all the delegates' documentation that the ListWheelScrollView will delegate to these for building its children, just like you did in the description of the abstract class.

Reply:
fixed.